css/stiles.css

```diff
@@ -0,0 +1,195 @@
+/*
```

Comment:
And rename `stiles.css` ===> `styles.css`

Reply:
Its my special "stile" :-)